tests/build_system_utils/Makefile

```diff
@@ -5,29 +5,77 @@ include $(RIOTBASE)/Makefile.include
 
 
 # Test utils commands
+
+ok_message = $(COLOR_ECHO) "$(COLOR_GREEN) [OK] $1 $(COLOR_RESET)"
+fail_message = $(COLOR_ECHO) "$(COLOR_RED) [FAIL] $1 $(COLOR_RESET)"
+
 define command_should_fail
-$1 2>/dev/null && { echo "Command '$1' should have failed but did not" >&2; $1; exit 1; } || true
+$1 2>/dev/null && { $(call fail_message,Command '$1' should have failed but did not) >&2; $1; exit 1; } || $(call ok_message,$2)
```

Comment:
This does not work, none of the `negative` functions detect when something does not fail.

Reply:
With this the test still pass:

``` diff
diff --git a/makefiles/utils/test-checks.mk b/makefiles/utils/test-checks.mk
index 3ee960cfd..f99a3d12e 100644
--- a/makefiles/utils/test-checks.mk
+++ b/makefiles/utils/test-checks.mk
@@ -4,7 +4,7 @@ test-ensure_value:
        @test "$@" = "$(call ensure_value,$@,"This should not fail")"

 test-ensure_value-negative:
-       @echo $(call ensure_value,$^,"This should fail")
+       true

 test-assert:
        @test -z "$(call assert,$@,"This should not fail")"
```